apps/transport/lib/transport_web/templates/dataset/details.html.heex

```diff
@@ -247,17 +247,22 @@
       <div class="dataset__logo">
         <%= img_tag(DB.Dataset.full_logo(@dataset), alt: @dataset.custom_title) %>
       </div>
-      <div>
-        <i class="icon fa fa-map-marker-alt" /><%= Dataset.get_territory_or_nil(@dataset) %>
-      </div>
       <div class="pt-12">
         <span class="dataset-metas-info-title"><%= dgettext("page-dataset-details", "Data published by") %></span>
-        <div class="dataset-type-info">
+        <div>
           <b>
             <%= link(@dataset.organization, to: dataset_path(@conn, :index, organization_id: @dataset.organization_id)) %>
           </b>
         </div>
       </div>
+      <%= if Enum.count(@dataset.legal_owners_aom) > 0 || Enum.count(@dataset.legal_owners_region) > 0 do %>
+        <div class="pt-12">
+          <span class="dataset-metas-info-title"><%= dgettext("page-dataset-details", "Data owned by") %></span>
```

Comment:
Ce n'est pas vraiment une notion de propriété mais bien de responsabilité. Voir pour une traduction plus fidèle en anglais ? Under the responsibility of ?